app/controllers/display_users_controller.rb

```diff
@@ -24,13 +24,25 @@ def search(query)
 
   protected
 
+    def sort_value
+      sort = params[:sort].blank? ? "name" : params[:sort]
+      case sort
+      when 'name'
+        'last_name'
+      when 'name desc'
+        'last_name DESC'
+      else
+        sort
```

Comment:
@scherztc Our tests never cover the `else` case.  So line 35 is untested.  However, I don't think we have a real-world situation where `sort` is anything but 'name' or 'name desc'.  So I think the best thing to do is remove the else condition from the method.  If you want to keep the else statement, we should create a controller spec for #sort_value that verifies `sort_value` returns `params[:sort]` when `params[:sort]` is something other than "name".